apps/nextjs-website/react-components/components/Editorial/Editorial.tsx

```diff
@@ -0,0 +1,77 @@
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import ContainerRC from '../common/ContainerRC';
+import { EditorialProps } from '../../types/Editorial/Editorial.types'
+import { useIsMobile } from './Editorial.helpers';
+import { BackgroundColor } from '../common/Common.helpers';
+import { Content as EditorialContent } from './Content';
+import { Ctas as EditorialCtas } from './Ctas';
+import { Image as EditorialImage } from './Image';
+
+const Editorial = (props: EditorialProps) => {
+  const {
+    image,
+    eyelet = '',
+    title,
+    body,
+    theme,
+    ctaButtons = [],
+    storeButtons = {},
```

Comment:
These default values seem unnecessary since they're optional parameters for `EditorialCtas` and the case for them being undefined is handled.